packages/e2e-test-utils-playwright/src/test.ts

```diff
@@ -140,6 +151,30 @@ const test = base.extend<
 		},
 		{ scope: 'worker', auto: true },
 	],
+	// Spins up a new browser for use by the Lighthouse fixture
+	// so that Lighthouse can connect to the debugging port.
+	// As a worker-scoped fixture, this will only launch 1
+	// instance for the whole test worker, so multiple tests
+	// will share the same instance with the same port.
+	lighthousePort: [
+		async ( {}, use ) => {
+			const port = await getPort();
+			const browser = await chromium.launch( {
+				args: [ `--remote-debugging-port=${ port }` ],
+			} );
+
+			await use( port );
+
+			await browser.close();
+		},
+		{ scope: 'worker' },
```

Comment:
Does it always launch the chromium even when we're not calling the `lighthouse` fixture? It seems wasteful but I'm not sure if it matters 😅 . I wonder if we really need to run the report in the same instance. I guess it can be considered an anti-pattern to run multiple lighthouse audits in parallel, so we don't need to worry about race conditions. Maybe it's okay to launch a new instance every time it's called? WDYT?

Reply:
Since it‘s a worker fixture this should launch one instance per worker, even if not used, but let me diuble check!

All of the performance tests will always run in sequence, no Lighthouse runs are ever done in parallel.

I left a comment about creating s new instance every time too, my only concern would be that it might be slower, but I haven‘t measured it yet.

Reply:
Clarification: **nothing** happens if `lighthouse` is not used in a test.

Reply:
Nice! Thanks for confirming! I'll take another look at this PR next week, but I think this is looking pretty good! Feel free to ping me again in case I forget 😆